php/src/Docker/DockerActionManager.php

```diff
@@ -897,11 +902,28 @@ public function ConnectMasterContainerToNetwork() : void
         $this->ConnectContainerIdToNetwork('nextcloud-aio-mastercontainer', '');
         // Don't disconnect here since it slows down the initial login by a lot. Is getting done during cron.sh instead.
         // $this->DisconnectContainerFromBridgeNetwork('nextcloud-aio-mastercontainer');
+
+        $apacheAdditionalNetwork = $this->configurationManager->GetApacheAdditionalNetwork();
+        if ($apacheAdditionalNetwork !== '') {
+            $this->ConnectContainerIdToNetwork('nextcloud-aio-mastercontainer', '', $apacheAdditionalNetwork, false);
```

Comment:
Shouldnt this rather be done via the compose.yaml itself I am not sure why the mastercontainer should also be inside the additonal network? what is the benefit of that?